packages/react-composites/src/mocks/useFakeChatAdapters.ts

```diff
@@ -96,8 +93,7 @@ const initializeAdapters = async (
 ): Promise<ChatAdapter[]> => {
   const remoteAdapters: ChatAdapter[] = [];
   for (const participant of participants) {
-    /* @conditional-compile-remove(communication-common-beta-v3) */
-    if (isMicrosoftBotIdentifier(participant.id)) {
+    if (((participant.id as CommunicationIdentifierKind).kind as string) === 'microsoftTeamsApp') {
```

Comment:
shouldn't we call `getIdentifierKind` here instead rather than casting it?
<img width="588" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/7634ebcb-5e7b-4b30-91a3-bed20153a68d">